.gitattributes

```diff
@@ -5,3 +5,5 @@
 # when the heading is exactly 7 characters long.
 *.md conflict-marker-size=100
 *.txt conflict-marker-size=100
+*.c text eof=lf
```

Comment:
This looks unrelated. Please provide those files in a separate PR if really required on your side.

Reply:
I had to do this to get LF after every line. Please do let me know if there is any other way of getting rid of them. Thanks!